### PR TITLE
Added placeholders js files

### PR DESCRIPTION
**Pull request by karminkarmen**
https://trello.com/c/3yONqHWK/368-header-end-of-body-js-placeholder-file

Added empty js placeholders files to input js content.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" xmlns="http://www.w3.org/1999/html">
```
Comment (apepper) [inline public/index.html]: Please revert this line to the original line.

File: public/index.html [hunk 2]
```diff
@@ -15,7 +16,7 @@
   <div id="application">
     <span class="loader">Loading...</span>
   </div>
-
   <script src="/index.js"></script>
-</body>
+  <script src="custom_js_before_body_end.js"></script>
+  </body>
```
Comment (apepper) [inline public/index.html]: Please revert this line (the `</body>` part) to the original line.

File: public/index.html [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
-<html lang="en">
+<html lang="en" xmlns="http://www.w3.org/1999/html">
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="generator" content="Scrivito by Infopark AG (scrivito.com)">
 
   <title>Scrivito Example App JS</title>
+  <script src="custom_js_head.js"></script>
```
Comment (apepper) [inline public/index.html]: I'm not 100% satisfied with the naming of the files. How about this?

* `js_snippets_head.js`
* `js_snippets_before_body_end.js`

**Comment by karminkarmen** [comment]
REVIEW (ed7e385) /cc @apepper changed information in snippet files

File: public/index.html [hunk 4]
```diff
@@ -7,6 +7,7 @@
   <meta name="generator" content="Scrivito by Infopark AG (scrivito.com)">
 
   <title>Scrivito Example App JS</title>
+  <script src="js_snippets_head.js"></script>
```
Comment (apepper) [inline public/index.html]: Can you move this at the very bottom of the `head`? Like this?

```html
  <script src="js_snippets_head.js"></script>
</head>
```

**Comment by karminkarmen** [comment]
REVIEW (8147bb1) /cc @apepper

**Comment by karminkarmen** [comment]
REVIEW (5e1e660) /cc @apepper

**Pull request merged**